interface/src/app/gene/network/network.js

```diff
@@ -221,6 +221,22 @@ angular.module('adage.gene.network', [
             g.query = true;
           }
         });
+
+        // format a list of genes for download
+        if (self.geneDownload !== null && self.geneDownload !== undefined) {
+          // release a previously-created Blob
+          URL.revokeObjectURL(self.geneDownload);
+        }
+        self.geneDownload = URL.createObjectURL(new Blob(
+          genes.map(function(g, index) {
+            var geneStr = g.label + ',' + g.entrezid + '\n';
```

Comment:
it looks like we're using Babel? If so, we can probably just use a template literal for this.

Reply:
That is a nice thought. We have incorporated Babel for a couple specific libraries that were written in ES6, but everything else in the codebase is ES5 and the build process does not currently support piping through Babel in a general way. (As an aside: I'd love to be using a more current build methodology NOW, but it will be a substantial project to retool. We have several steps planned that will take us in the right direction, but it is not the top priority.)